core/src/main/scala/kafka/server/ConfigHandler.scala

```diff
@@ -19,10 +19,10 @@ package kafka.server
 
 import java.util.Properties
 
+import DynamicConfigs.Broker._
```

Comment:
Would it perhaps be better to have this import in the `BrokerConfigHandler` class?